src/verilog/verilog_elaborate.cpp

```diff
@@ -611,7 +611,7 @@ void verilog_typecheckt::collect_symbols(const verilog_lett &let)
 
 void verilog_typecheckt::collect_symbols(const verilog_statementt &statement)
 {
-  if(statement.id() == ID_assert || statement.id() == ID_assume)
+  if(statement.id() == ID_verilog_immediate_assert)
```

Comment:
Coming! This is a bigger change, since assumptions aren't symbols yet. They simply get thrown into the transition system as constraint, which doesn't fit once you have temporal logic in the assumption.